config/serverless.yml

```diff
@@ -10,6 +10,8 @@ xpack.fleet.internal.retrySetupOnBoot: true
 xpack.fleet.internal.useMeteringApi: true
 xpack.searchSynonyms.enabled: false
 
+xpack.searchQueryRules.enabled: false
```

Comment:
If we add this here then I don't _think_ we need to add the same to `config/serverless.security.yml` & `config/serverless.oblt.yml` since we merge those config files.